apps/jira/jira-app/package.json

```diff
@@ -3,27 +3,28 @@
   "version": "1.1.0",
   "private": true,
   "devDependencies": {
-    "@babel/core": "7.3.4",
-    "@babel/plugin-proposal-class-properties": "7.3.4",
-    "@babel/plugin-transform-runtime": "7.3.4",
-    "@babel/preset-env": "7.3.4",
-    "@babel/preset-react": "7.0.0",
+    "@babel/core": "7.8.4",
+    "@babel/plugin-proposal-class-properties": "7.8.3",
+    "@babel/plugin-transform-runtime": "7.8.3",
+    "@babel/preset-env": "^7.8.7",
+    "@babel/preset-react": "7.8.3",
     "@contentful/contentful-extension-scripts": "^0.19.1",
-    "@contentful/eslint-config-extension": "0.2.0",
+    "@contentful/eslint-config-extension": "^0.2.0",
     "@testing-library/react": "8.0.4",
     "@types/jest": "24.0.15",
     "@types/lodash.get": "^4.4.6",
     "@types/react": "^16.8.17",
     "@types/react-dom": "^16.8.4",
     "@types/webpack-env": "1.13.9",
+    "@typescript-eslint/parser": "^4.14.2",
     "contentful-cli": "1.2.12",
     "cssnano": "4.1.10",
     "eslint": "^6.0.1",
     "fetch-mock": "^8.3.2",
     "node-fetch": "^2.6.1",
     "prettier": "1.19.1",
     "sass": "^1.23.7",
-    "typescript": "3.5.2"
+    "typescript": "^3.9.7"
```

Comment:
As TypeScript doesn't follow semantic versioning, we can go to the latest version straight away - unless you encountered any issues.

Reply:
I was just using the latest 3.X version. But yes that is a good point :) I will update to the latest version